src/debug.ts

```diff
@@ -0,0 +1,51 @@
+/*
+  TODO:
+  import DFP from './networks/DFP';
+*/
+
+import Ad from './Ad';
+import Page from './Page';
+import AutoRefresh from './plugins/AutoRefresh';
+import AutoRender from './plugins/AutoRender';
+import DeveloperTools from './plugins/DeveloperTools';
+import Responsive from './plugins/Responsive';
+import Sticky from './plugins/Sticky';
+import { IPlugin } from './types';
+
+const plugins: { [key: string]: IPlugin } = {
+  AutoRefresh,
+  AutoRender,
+  DeveloperTools,
+  Responsive,
+  Sticky,
+};
+
+const pluginKeys = Object.keys(plugins);
+
+pluginKeys.forEach((plugin) => {
+  if (!(window as any)[`_ADJS`].Plugins[plugin]) {
```

Comment:
This doesn’t look correct. Can you verify? I think we should always be pushing the production developer tools in. Make sure you do an isServer check